### ISSUE TITLE
Missing required argument $name of Nosto\Tagging\Logger\Logger

### ISSUE THREAD

**Issue by JacobDrummond** [Nosto/nosto-magento2#161]
### Environment
* Windows 10
* Apache 2.4.17
* MySQL  5.0.11-dev
* PHP 5.6.15
* Magento 2.1.8
* Nosto_Tagging 2.3.0

### Replication steps
1. `composer update update nosto/*`
1. `rm -rf var/*/`
1. `rm -rf pub/static/*`
1. `rm -rf var/view_preprocessed/*`
1. Load a category with products

### Expected result
* Category page is rendered

### Actual result
* `Exception #0 (BadMethodCallException): Missing required argument $name of Nosto\Tagging\Logger\Logger.`
![nosto-error](https://user-images.githubusercontent.com/8377620/29567412-052dc93a-8745-11e7-9933-df9ccbec1357.png)

### Cause
* No `<arguments>` node in the [`di.xml`](/Nosto/nosto-magento2/blob/2.3.0/etc/di.xml) logger definition. 
* Basically `Nost_Tagging` is missing step 3 from [this tutorial](https://magento.stackexchange.com/a/75954).
* See, for example, Magento2's default [logger definition](/magento/magento2/blob/2.1.8/app/etc/di.xml#L181-L189).

**Comment by hannunosto** [comment]
Thanks for reporting this. This is now fixed in release 2.3.1 (https://github.com/Nosto/nosto-magento2/releases/tag/2.3.1).
